portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,15 +40,20 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     String quantityChosen = request.getParameter("number");
-
+    String commentOrder = request.getParameter("order");
     int amount;
     try {
         amount = Integer.parseInt(quantityChosen);
     } catch (NumberFormatException e) {
         amount = 1;
     }
 
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    Query query;
+    if (commentOrder != null && commentOrder.equals("newest")){
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
Space before the braces
`("newest")) {`